compose/compose-file/compose-versioning.md

```diff
@@ -121,9 +121,24 @@ discoverable at a hostname that's the same as the service name. This means
 [links](compose-file-v2.md#links) are largely unnecessary. For more details, see
 [Networking in Compose](compose-file-v2.md#networking.md).
 
+> **Note**: When specifying the Compose file version to use, make sure to
```

Comment:
This is a much-needed update, @thaJeztah . Thank you for taking it on. Before I merge, please have a look at my minor stylistic and text edits.

Reply:
Yes, I kept seeing people being confused (or making this mistake due to copy/pasting old examples), so thought to call it out explicitly 🤗 

Saves me some time as well in the long run explaining it 😂

Reply:
Done. Have a look at the final change. Ok to squash and merge this one, @thaJeztah ?